lib/flood_risk_engine/engine.rb

```diff
@@ -4,6 +4,7 @@
 require "high_voltage"
 require "ea/address_lookup"
 require "defra_ruby/area"
+require "defra_ruby_email"
```

Comment:
Is there a reason we have `defra_ruby_email` and not `defra_ruby/email` like we do for our other gems?

Reply:
There seems to be a difference in how namespacing is handled in an engine compared to a plain old gem.

I couldn't get to the bottom of it, so fell back to what rails generates. We had the same issue in [defra-ruby-mocks](https://github.com/DEFRA/defra-ruby-mocks) hence it too has a `defra_ruby_mocks` namespace.

Reply:
Ahh, interesting, I didn't know that! Good to know 👍